meos/src/general/temporal.c

```diff
@@ -1821,7 +1821,9 @@ temporal_interpolation(const Temporal *temp)
   char *result = palloc(sizeof(char) * MOBDB_INTERPOLATION_STR_MAXLEN);
   assert(temptype_subtype(temp->subtype));
   interpType interp = MOBDB_FLAGS_GET_INTERP(temp->flags);
-  if (temp->subtype == TINSTANT || interp == DISCRETE)
+  if (temp->subtype == TINSTANT)
+    strcpy(result, "None");
+  else if (interp == DISCRETE)
```

Comment:
I thought we were renaming DISCRETE to NONE.
What is here the difference between "None" and "Discrete"?